.github/workflows/packages.yml

```diff
@@ -268,7 +268,7 @@ jobs:
     concurrency: ${{ github.workflow }}
     permissions:
       contents: read
-    if: github.event_name != 'pull_request' && github.ref != 'refs/heads/dev' && !startsWith(github.ref, 'refs/heads/dev/')
+    if: github.repository == 'termux/termux-packages'
```

Comment:
Picking back up from some of my thoughts in #21906,
how would this affect forks?

My best guess would be it's only applicable to them if they use the `packages.yml` workflow to automate uploading CI built packages to their repos.
In which case they'd need to modify it somewhat anyways.